lib/decoder/real-time.js

```diff
@@ -48,10 +48,11 @@ exports.decode = async function (buffer) {
   const time = Math.round(parseFloat(data.timestamp) * 1000)
 
   // uri stem includes query params (ex: /123/<guid>/<digest>/file.mp3?le=<le>)
-  const le = url.parse(data['cs-uri-stem'], true).query.le
+  const parsed = url.parse(data['cs-uri-stem'] || '', true)
+  const le = parsed.query.le
 
-  // but digest is always 2nd to last
-  const digest = (data['cs-uri-stem'] || '').split('/').slice(-2, -1)[0]
+  // but digest is always 2nd to last (trimming trailing slashes)
+  const digest = parsed.pathname.replace(/\/+$/, '').split('/').slice(-2, -1)[0]
```

Comment:
So 2 changes: use the parsed `pathname` here, which won't include query params at all.  So if somehow the user added a `/` within/after the query params, it won't matter here.

Also trims any trailing `/` from the path.  So we're always correctly counting back to the 2nd-to-last digest.